### PR TITLE
Feature/SEM-021

### PR DESCRIPTION
**Pull request by LuisPerez007**
#Descrption
#https://ditmar.github.io/backoffice-openscience/branch-SEM-021/?path=/story/ui-components-atoms-toggle--default-enabled

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "@storybook/react": "8.2.9",
     "@storybook/react-vite": "8.2.9",
     "@testing-library/dom": "10.4.0",
-    "@testing-library/jest-dom": "6.4.5",
+    "@testing-library/jest-dom": "6.5.0",
```
Comment (Ditmar) [inline package.json:52]: why are you changing the version of jest ?

File: src/ui/components/atoms/DangerButton/Danger-Button.scss
Comment (Ditmar) [inline src/ui/components/atoms/DangerButton/Danger-Button.scss:1]: why are you delete it these files

File: src/ui/components/atoms/Toggle.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  const handleToggle = () => {
+    if (!disabled) {
+      setChecked(!checked);
+      if (onChange) onChange(!checked);
+    }
+  };
+
+  return (
+    <div className={`${styles.container} ${disabled ? styles.disabled : ''}`}>
```
Comment (Ditmar) [inline src/ui/components/atoms/Toggle.tsx:21]: please use this approach https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments

File: package.json (hunk 1 above)
Comment (LuisPerez007) [reply package.json:52]: sorry Ing, I corrected it

File: src/ui/components/atoms/DangerButton/Danger-Button.scss
Comment (LuisPerez007) [reply src/ui/components/atoms/DangerButton/Danger-Button.scss:1]: sorry Ing I thought someone had the wrong branch

**Review by Ditmar — APPROVED** [review]

**Pull request merged**